spec/integration/common.js

```diff
@@ -59,16 +63,27 @@ Given('editor is open', () => {
   cy.wait(2000);
 });
 
+Given('add workflow popup is open', () => {
+  click('Add Workflow Button');
+});
+
```

Comment:
this rule belongs to workflows test case. Can you move it to workflows/steps.js?